src/analysis/special_functions/arsinh.lean

```diff
@@ -0,0 +1,169 @@
+/-
+Copyright (c) 2020 James Arthur. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: James Arthur, Chris Hughes, Shing Tak Lam.
+-/
+import analysis.special_functions.trigonometric
+import data.real.basic
+import analysis.special_functions.pow
+
+open real
+local attribute [pp_nodot] real.log
+noncomputable theory
+
+/-!
+# Inverse of the sinh function
+
+In this file we prove that sinh is bijective and hence has an
+inverse, arsinh.
+
+## Main Results
+
+- `sinm_injective`: The proof that `sinm` is injective
+- `sinm_surjective`: The proof that `sinm` is surjective
+- `sinm_bijective`: The proof `sinm` is bijective
+
+## Notation
+
+- `arsinh`: The inverse function of `sinm`
+
+## Tags
+
+arsinh, sinh injective, sinh bijective, sinh surjective
+-/
+
+
+/-- `arsinh` is defined using a logarithm-/
```

Comment:
```suggestion
/-- `arsinh` is defined using a logarithm -/
```

Reply:
Actually, please copy the full definition into the docstring. My analysis is rusty, so I wouldn't mind being able to take a peek at the definition by hovering over the name.

Reply:
OK, I will just make this in one push. Thankyou both in your suggestions

Reply:
OK! Will do now.